services/admin/src/store.ts

```diff
@@ -489,4 +536,276 @@ export class Store {
       );
     });
   }
+
+  /**
+   * Gets a summary of the write-in adjudication status.
+   */
+  getWriteInAdjudicationSummary({
```

Comment:
(for a future pr) it might be useful for reporting to get a summary that is just the adjudicated values & counts, without the level of detail of the transcribed values, but the frontend could also just figure that out from this API